apps/health_alerts/static/js/healthAlertPriorityFilter.js

```diff
@@ -0,0 +1,64 @@
+export default function() {
+  const selectElement = document.querySelector('.select-priority');
+
+  if (selectElement) {
+    // add a handler for the change event for the priority select dropdown
+    selectElement.addEventListener('change', (event) => {
+      const selectedPriority = event.target.value;
+      const allRows = document.querySelectorAll("[data-priority]");
+      // we want to show any row that has the selectedPriority
+      let rowsToShow = document.querySelectorAll(`[data-priority~="${selectedPriority}"]`);
+      if (selectedPriority === 'All') {
+        // show all rows again
+        rowsToShow = allRows;
+      }
+      // filter the rows, restripe them, and then filter the right side links
```

Comment:
If `rowsToShow` is falsy, can we show a hidden div that has the text `No Health Alerts Found` or something like that?

Reply:
Also if possible, can we also hide the right scroll nav when rowstoShow is empty. When rowsToShow is Falsy, the nav headings are not visible, but the left border of the rightScrollNav is visible.